### PR TITLE
User Profile Phone Number

### PR DESCRIPTION
**Pull request by jah3**

### CODE REVIEW

File: src/main/java/md/maib/entity/Customer.java [hunk 1]
```diff
@@ -1,9 +1,12 @@
 package md.maib.entity;
 
 import jakarta.persistence.*;
+
```
Comment (taragrammatonn) [inline src/main/java/md/maib/entity/Customer.java:4]: remove empty space

File: src/main/java/md/maib/entity/Customer.java [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Objects;
 
+
```
Comment (taragrammatonn) [inline src/main/java/md/maib/entity/Customer.java:9]: remove empty space

File: src/main/java/md/maib/entity/PhoneNumber.java [hunk 3]
```diff
@@ -0,0 +1,88 @@
+package md.maib.entity;
+
+import jakarta.persistence.*;
+
```
Comment (taragrammatonn) [inline src/main/java/md/maib/entity/PhoneNumber.java:4]: use idea code format tool

File: src/main/java/md/maib/repository/CustomerDAO.java [hunk 4]
```diff
@@ -81,7 +120,8 @@ public Customer updateCustomerById(Long id, Customer customerDetails) {
             pstmt.setString(2, customerDetails.getLastName());
```
Comment (taragrammatonn) [inline src/main/java/md/maib/repository/CustomerDAO.java:120]: why you use here some constant ids?

File: src/main/java/md/maib/entity/Customer.java [hunk 5]
```diff
@@ -25,15 +28,22 @@ public class Customer {
     @Column(name = "pan", nullable = false, unique = true, length = 16)
     private String pan;
 
+    @Column(name = "age", nullable = false)
```
Comment (taragrammatonn) [inline src/main/java/md/maib/entity/Customer.java:31]: why age in phone changes commit?

File: src/main/java/md/maib/entity/Customer.java [hunk 6]
```diff
@@ -25,15 +28,22 @@ public class Customer {
...
+    @OneToMany(mappedBy = "customer", cascade = CascadeType.ALL, fetch = FetchType.EAGER)
+    private List<PhoneNumber> phoneNumbers;
+
     public static class Builder {
         private Long id;
         private String firstName;
         private String lastName;
         private String pan;
         private String cvv;
+        private int age;
```
Comment (taragrammatonn) [inline src/main/java/md/maib/entity/Customer.java:46]: decompose all age changes into another commit

File: src/main/java/md/maib/entity/Transaction.java [hunk 7]
```diff
@@ -18,6 +18,14 @@ public class Transaction {
     public Transaction() {
     }
 
+    public Transaction(Long transactionId, Customer customer, BigDecimal amount, LocalDateTime transactionDate) {
```
Comment (taragrammatonn) [inline src/main/java/md/maib/entity/Transaction.java:21]: why this changes here?

File: src/main/java/md/maib/repository/CustomerDAO.java [hunk 8]
```diff
@@ -22,28 +26,62 @@ public class CustomerDAO {
     private static final Logger LOGGER = LoggerFactory.getLogger(CustomerDAO.class);
 
     public Optional<Customer> getCustomerById(Long id) {
-        String SQL = "SELECT * FROM users WHERE user_id = ?";
+        String SQL = "SELECT u.*, pn.phone_id, pn.phone_number, t.transaction_id, t.amount, t.transaction_date " +
+                "FROM users u " +
+                "LEFT JOIN phone_numbers pn ON u.user_id = pn.user_id " +
+                "LEFT JOIN transactions t ON u.user_id = t.user_id " +
```
Comment (taragrammatonn) [inline src/main/java/md/maib/repository/CustomerDAO.java:32]: same, in this pro should be only phone changes

**Review by taragrammatonn — CHANGES_REQUESTED** [review]

File: src/test/resources/schema_test.sql [hunk 9]
```diff
@@ -1,16 +1,34 @@
 CREATE TABLE  IF NOT EXISTS users (
-                       user_id SERIAL PRIMARY KEY,
-                       first_name VARCHAR(50) NOT NULL,
-                       last_name VARCHAR(50) NOT NULL,
-                       pan VARCHAR(16) UNIQUE NOT NULL,
-                       cvv VARCHAR(3) NOT NULL
-);
+                                      user_id SERIAL PRIMARY KEY,
```
Comment (taragrammatonn) [inline src/test/resources/schema_test.sql:2]: form sql queries like this
CREATE TABLE  IF NOT EXISTS users (
user_id      SERIAL PRIMARY KEY,
first_name VARCHAR(50) NOT NULL,
last_name  VARCHAR(50) NOT NULL,
pan            VARCHAR(16) UNIQUE NOT NULL,
cvv            VARCHAR(3) NOT NULL
);

File: src/main/java/md/maib/repository/CustomerDAO.java (hunk 4 above)
Comment (jah3) [reply src/main/java/md/maib/repository/CustomerDAO.java:120]: those are parameter index
